docs/docs/metrics.md

```diff
@@ -98,23 +98,22 @@ Dashes in metric names are replaced with underscores.
   a deployment might be dismissed by Marathon, when there are too many
   concurrent deployments.
 * `marathon.groups.active.gauge` — the number of active groups.
-* `marathon.leadership.duration.counter.seconds` — the duration of
+* `marathon.leadership.duration.gauge.seconds` — the duration of
   current leadership.
-* `marathon.persistence.gc.compaction-duration.counter.seconds` — the
-  total time spent in the Marathon GC compaction phase by the current
-  instance.
 * `marathon.persistence.gc.runs.counter` — the count of Marathon GC runs
   since it became a leader.
-* `marathon.persistence.gc.scanning-duration.counter.seconds` — the
-  total time spend in the Marathon GC scanning phase by the current
-  instance.
+* `marathon.persistence.gc.compaction.duration.timer.seconds` —
+  a histogram of Marathon GC compaction phase durations, and a meter for
+  compaction durations.
+* `marathon.persistence.gc.scan.duration.timer.seconds` — a histogram of
+  Marathon GC scan phase durations, and a meter for scan durations.
 * `marathon.tasks.launched.counter` — the count of tasks launched by
   the current Marathon instance since it became a leader.
 * `marathon.tasks.running.gauge` — the number of running tasks at the
   moment.
 * `marathon.tasks.staged.gauge` — the number of tasks staged at the
   moment.
-* `marathon.uptime.counter.seconds` — uptime of the current Marathon
+* `marathon.uptime.gauge.seconds` — uptime of the current Marathon
```

Comment:
there is a mixture of `timer.seconds` and `gauge.seconds`   It is unclear to me as to when.
I also am challenged by an always increasing gauge.

Reply:
Well, I am not too crazy about it either, but if to choose from a set of tradeoffs, I'd prefer this one.